### PR TITLE
Add swss events to test_events

### PR DESCRIPTION
**Pull request by zbud-msft**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

#### What is the motivation for this PR?

Add if_state, pfc_storm, chk_crm_threshold swss test to nightly test

#### How did you do it?

Added new test case for test_events

#### How did you verify/test it?

Pipeline, manual testing

### CODE REVIEW

File: tests/telemetry/events/swss_events.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    assert ret["rc"] == 0, "Failing to shutdown interface Ethernet0"
+
+    ret = duthost.shell("config interface startup Ethernet0")
+    assert ret["rc"] == 0, "Failing to startup interface Ethernet0"
+
+
+def generate_pfc_storm(duthost):
+    logger.info("Generating pfc storm on Ethernet4:4")
+    queue_oid = duthost.get_queue_oid("Ethernet4", 4)
+    duthost.shell("redis-cli -n 2 hset \"COUNTERS:{}\" \"DEBUG_STORM\" \"enabled\"".format(queue_oid))
```
Comment (qiluo-msft) [inline tests/telemetry/events/swss_events.py:35]: Prefer sonic-db-cli than redis-cli.

Comment (zbud-msft) [reply tests/telemetry/events/swss_events.py:35]: I will change

Comment (zbud-msft) [reply tests/telemetry/events/swss_events.py]: Changed to sonic-db-cli

**Review by qiluo-msft — DISMISSED** [review]

**Review by prabhataravind — DISMISSED** [review]
lgtm

File: tests/telemetry/events/swss_events.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    ret = duthost.shell("config interface startup {}".format(IF_STATE_TEST_PORT))
+    assert ret["rc"] == 0, "Failing to startup interface {}".format(IF_STATE_TEST_PORT)
+
+
+def generate_pfc_storm(duthost):
+    logger.info("Generating pfc storm")
+    queue_oid = duthost.get_queue_oid(PFC_STORM_TEST_PORT, PFC_STORM_TEST_QUEUE)
+    duthost.shell("sonic-db-cli COUNTERS_DB HSET \"COUNTERS:{}\" \"DEBUG_STORM\" \"enabled\"".
+                  format(queue_oid))
+    duthost.shell("pfcwd start --action alert {} {} --restoration-time {}".
```
Comment (neethajohn) [inline tests/telemetry/events/swss_events.py:50]: please use '--action drop' to mimic real use case

Comment (zbud-msft) [reply tests/telemetry/events/swss_events.py]: Done

**Review by neethajohn — APPROVED** [review]

**Comment by wangxin** [comment]
This PR was merged before 202311 branch was created from master branch. No need to cherry-pick for 202311 branch.

**Pull request merged**